README.md

```diff
@@ -61,26 +61,32 @@ This contains the id of the issue and the error.
 ### CreateIssue command
 This command creates a Jira issue.
 #### Input
-This commands input is the project the issue should be created under, the issue type and the title.
+This command inputs are the project that the issue should be created under, the issue type and the summary (title).
 ```
-"input": {
-    "project": "TEST",
-    "issue_type": "Story",
-    "title": "Fix csetcd bug"
+"fields": {
+    "project":  
+       {
+          "key": "TEST"
+       },
+    "issuetype": 
+       {
+          "name": "Story"
+       },
+    "summary": "Fix csetcd bug"
```

Comment:
Real heroes don't wear capes, they update the docs 🙏  thank you Yaser this looks better and more intuitive.